backend/sample/test.txt

```diff
@@ -0,0 +1,37 @@
+# test taxonomy

```

Comment:
Kudos for having wrote this test. This will be usefull to @BryanH01 also, I guess.